README.md

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2011 Autonomous Robots and Cognitive Systems Laboratory, Universidad de Costa Rica
+Copyright (c) 2016-2018 Autonomous Robots and Cognitive Systems Laboratory, Universidad de Costa Rica
```

Comment:
Sometimes this reads  Autonomous Robots and Cognitive Systems Laboratory, Universidad de Costa Rica and others  Autonomous Robots and Cognitive Systems Laboratory; without the Universidad de Costa Rica. Standarize to one or the other please.